specs/latest/1.0/index.html

```diff
@@ -3481,6 +3481,14 @@ <h2 class="no-toc">Table of contents</h2>
 
 </p>
 
+    <h3>No Shader Binaries</h3>
+
+<p>
+
+Accessing binary representations of compiled shaders is not supported in the WebGL API. This includes OpenGL ES 2.0 ShaderBinary entry point. In addition, querying shader binary formats and shader compiler with getParameter is not supported in the WebGL API.
```

Comment:
Can we rephrase this a little? Also, could you please wrap it to 100 characters?

"This includes OpenGL ES" -> "This includes the OpenGL ES"
"and shader compiler with" -> "and the availability of a shader compiler via"

Suggest adding:

"All WebGL implementations must implicitly support an on-line shader compiler."

Thanks.